blocksync/blocksync.go

```diff
@@ -330,13 +330,14 @@ func (bs *blockSyncer) syncStageChecker() {
 func (bs *blockSyncer) SyncStatus() (uint64, uint64, uint64, string) {
 	var syncSpeedDesc string
 	syncBlockIncrease := atomic.LoadUint64(&bs.syncBlockIncrease)
+	increase := syncBlockIncrease
```

Comment:
i didn't comprehend the cause of this issue, nor the reason of this modification to resolve it

Reply:
i don't quite get it either, `syncBlockIncrease` is a local var, same as the new `increase` you added, can you dig to find out the root-cause?

Reply:
sorry, it's my fault. I will test and dig it